si/orange3-imageanalytics.jaml

```diff
@@ -0,0 +1,497 @@
+            https://api.garaza.io/: false
+        def `is_local_embedder`:
+            is_local: false
+        def `_get_model_settings_confidently`:
+            "'{:s}' is not a valid model, should be one of: {:s}": "'{:s}' ni veljaven model, moral bi biti eden izmed: {:s}"
+            ', ': false
+        def `_init_embedder`:
+            batch_size: false
+            image: false
+            target_image_size: false
+        def `__call__`:
+            Please provide a column for image path: Prosimo, navedite stolpec za pot do slike
+        def `from_table`:
+            image: false
+        def `__enter__`:
+            ImageEmbedder: false
+        def `construct_output_data_table`:
+            n{:d}: false
+            hidden: false
+        def `prepare_output_data`:
+            Skipped images: Izpuščene slike
+            Embedded images: Vpete slike
+    __main__: false
+    tests/test_images/example_image_0.jpg: false
+image_grid.py:
+    class `ImageGrid`:
+        def `_reduce_dimensions`:
+            MDS: false
+            precomputed: false
+            PCA: false
+            TSNE: false
+            pca: false 
+        def `_get_assignments`:
+            sqeuclidean: false
+            lap: false
+import_images.py:
+    class `ImportImages`:
+        ImgData: false
+        path: false
+        format: false
+        height: false
+        width: false
+        size: false
+        ImgDataError: false
+        error: false
+        error_str: false
+        def `image_meta`:
+            *: false
+    def `scan`:
+        *: false
+        .*: false
+    def `image_meta_data`:
+        ascii: false
+        nan: false
+    def `create_table`:
+        category: kategorija
+        image name: ime slike
+        image: null
+        type: false
+        origin: false
+        size: velikost
+        width: širina
+        height: višina
+local_embedder.py:
+    class `LocalEmbedder`:
+        def `__init__`:
+            target_image_size: false
+            .ImageEmbedder.httpcache: false
+server_embedder.py:
+    class `ServerEmbedder`:
+        def `__init__`:
+            image/jpeg: false
+utils/embedder_utils.py:
+    class `ImageLoader`:
+        def `__init__`:
+            .ImageEmbedder.httpcache: false
+        def `load_image_or_none`:
+            RGB: false
+        def `load_image_bytes`:
+            JPEG: false
+        def `_load_image_from_url_or_local_path`:
+            http: false
+            https: false
+            Image skipped: Izpuščena slika
+            ftp: false
+            data: false
+utils/image_utils.py:
+    def `filter_image_attributes`:
+        type: false
+        image: false
+    def `extract_image_path`:
+        origin: false
+        http: false
+        https: false
+        ftp: false
+        data: false
+        /: false
+widgets/__init__.py:
+    Image Analytics: Analiza slik
+    Management and embedding of image data.: Upravljanje in vpetje slikovnih podatkov.
```

Comment:
Ko boš spreminjala to: mogoče "vložitve slikovnih podatkov" (ker bi očitnejši prevo, "vlaganje", zazvenela kot kisle kumarice).